src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLoggerProvider.cs

```diff
@@ -20,6 +20,9 @@ namespace Serilog.Extensions.Logging
     /// <summary>
     /// An <see cref="ILoggerProvider"/> that pipes events through Serilog.
     /// </summary>
+#if LOGGING_BUILDER
+    [ProviderAlias("Serilog")]
+#endif
```

Comment:
Just to help navigate this huge changeset, here's the first functional change...